packages/components/organisms/f-status-banner/src/constants.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
+export const BASE_HERO_URL = 'https://d30v2pzvrfyzpo.cloudfront.net/status-banner/';
```

Comment:
Will this CloudFront distribution always be the same for all locales?

Reply:
Yup the base url will and then the locale is pulled in [here](https://github.com/justeat/fozzie-components/pull/721/files#diff-2a00b3d793dfc8d90b89b326bc905d27bfa36ea2a328f813172b4b7ae59f2a6bR55).